interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
+
+// Should Doctrine run in "dev mode"?
+$GLOBALS["doctrine_dev_mode"] = false;
+
+// Should Doctrine make use of connection pooling?
+$GLOBALS["doctrine_connection_pooling"] = true;
```

Comment:
very cool.
If we are always going to set this to true, wouldn't it be better to just encapsulate it in the createConnection class.